### PR TITLE
Add CQL Test Queries to Troubleshooting

### PR DESCRIPTION
**Pull request by alexanderkiel**
Not sure about integrating the token fetching here. Perhaps @EmteZogaf you have an idea that also works for FLARE.

### CODE REVIEW

File: feasibility-triangle/troubleshooting/cql/5-Konsent.cql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+library "5-Konsent"
```
Comment (juliangruendner) [inline feasibility-triangle/troubleshooting/cql/5-Konsent.cql:1]: should not be called Konsent - instead Consent. - or German Einwilligung

File: feasibility-triangle/troubleshooting/cql/report.jq [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (juliangruendner) [inline feasibility-triangle/troubleshooting/cql/report.jq:1]: This does not work on my version of blaze 0.27.1
as the response from blazectl (OUTPUT in the test-cql.sh) looks as follows:

{"date":"2024-06-25T06:45:12.855811506Z","group":[{"population":[{"count":0,"code":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/measure-population","code":"initial-population"}]}}]}],"type":"summary","resourceType":"MeasureReport","measure":"urn:uuid:de9b855a-4415-44d4-ab56-f777331f85b4","extension":[{"url":"https://samply.github.io/blaze/fhir/StructureDefinition/eval-duration","valueQuantity":{"value":0.064653792,"unit":"s","system":"http://unitsofmeasure.org","code":"s"}}],"status":"complete","period":{"start":"1900","end":"2200"}}

File: feasibility-triangle/troubleshooting/test-cql.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
```
Comment (juliangruendner) [inline feasibility-triangle/troubleshooting/test-cql.sh:1]: there is no Readme on how to use the script in the repository neither is there an explanation in the script itself.
Either change the existing troubleshooting.md or add a help message.
Note, that someone troubleshooting should be made aware of the get-fhir-server-access-token.sh in the repository.

File: feasibility-triangle/troubleshooting/test-cql.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (juliangruendner) [inline feasibility-triangle/troubleshooting/test-cql.sh:2]: there should be a check and if blazectl and jq are available and if false an appropriate message should be displayed explaining to the user where to get it.
At the moment if one does not have blazectl it fails silently

File: feasibility-triangle/troubleshooting/cql/report.jq (hunk 2 above)
Comment (juliangruendner) [reply feasibility-triangle/troubleshooting/cql/report.jq:1]: note it works if i only leave the result: .... part of the file

**Review by juliangruendner — CHANGES_REQUESTED** [review]

File: feasibility-triangle/troubleshooting/cql/5-Konsent.cql (hunk 1 above)
Comment (alexanderkiel) [reply feasibility-triangle/troubleshooting/cql/5-Konsent.cql:1]: I used the naming from Confluence. Perhaps we should change it there also?

File: feasibility-triangle/troubleshooting/cql/report.jq (hunk 2 above)
Comment (alexanderkiel) [reply feasibility-triangle/troubleshooting/cql/report.jq:1]: Yes I wrote it only for the upcoming Blaze version with CQL cache. Now it works also for older versions.

File: feasibility-triangle/troubleshooting/test-cql.sh (hunk 4 above)
Comment (alexanderkiel) [reply feasibility-triangle/troubleshooting/test-cql.sh:2]: ok added checks for both

**Review by juliangruendner — APPROVED** [review]

**Pull request merged**